database/factories/GenreMovieFactory.php

```diff
@@ -24,12 +24,8 @@ class GenreMovieFactory extends Factory
     public function definition()
     {
         return [
-            'genre_id' => function() {
-                return Genre::factory()->create()->id;
-            },
-            'movie_id' => function() {
-                return Movie::factory()->create()->id;
-            },
+            'genre_id' => rand(1, 12),
```

Comment:
@vrodriguero Hi Vick, I decided to make this a random number between the id's of the genres since if I call the Genre::factory() it creates more than the genre we wish to have. What do you think?

Reply:
Hey Jayd,

I think that you have another option to fix this.

Try creating a Genre movie seeder then, and in there you can use eloquent to query a random movie and a random genre to assign the values.

So you have to create a GenreMovie Seeder and on the run() you can have something like

GenreMovie::factory()->create([
   'movie_id' => get random movie->id
   'genre_id' => get random genre->id
])

Take a look at this, I think it might help you 
[Collection](https://laravel.com/docs/8.x/collections#method-random)